### ISSUE TITLE
rocketqa在python3.10 paddle-gpu2.5.1 下报错

### ISSUE THREAD

**Issue by Mewral** [PaddlePaddle/RocketQA#113]
### 环境
python=3.10.0
paddlepaddle-gpu=2.5.1.post116
CUDA=12.0
rocketqa=1.1.0

### 报错截图
![image](https://github.com/PaddlePaddle/RocketQA/assets/29936932/7064d6ce-9809-4377-aeef-7c122cf7ce7d)

**Comment by Mewral** [comment]
将paddle-gpu版本改为2.3.2解决

**Comment by Jeffwan** [comment]
@Mewral Did you install `paddle-fleet`?

**Comment by Mewral** [comment]
@Jeffwan Yes, but paddle 2.3.2 has a different structure of fleet like 'paddle.fluid.xxx.fleet' not the same as 2.5.1's 'paddle.fluid.incubate.fleet'
